public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/networkInsightsDashboard.json

```diff
@@ -0,0 +1,2094 @@
+{
+    "__inputs": [
+    ],
+    "__elements": [],
+    "__requires": [
+      {
+        "type": "grafana",
+        "id": "grafana",
+        "name": "Grafana",
+        "version": "8.5.1"
+      },
+      {
+        "type": "datasource",
+        "id": "grafana-azure-monitor-datasource",
+        "name": "Azure Monitor",
+        "version": "1.0.0"
+      },
+      {
+        "type": "panel",
+        "id": "table",
+        "name": "Table",
+        "version": ""
+      },
+      {
+        "type": "panel",
+        "id": "timeseries",
+        "name": "Time series",
+        "version": ""
+      }
+    ],
+    "editable": true,
+    "id": null,
+    "links": [],
+    "liveNow": false,
+    "panels": [
+      {
+        "collapsed": true,
+        "datasource": {
+          "type": "datasource",
+          "uid": "grafana"
+        },
+        "gridPos": {
+          "h": 1,
+          "w": 24,
+          "x": 0,
+          "y": 0
+        },
+        "id": 12,
+        "panels": [
+          {
+            "datasource": {
+              "type": "grafana-azure-monitor-datasource",
+              "uid": "${DS_AZURE_MONITOR}"
```

Comment:
this should be ${ds} that's why the dashboard looks broken. It is pointing to a datasource name that does not exist